src/lib/y2network/autoinst_profile/networking_section.rb

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2network/autoinst_profile/routing_section"
+
+module Y2Network
+  module AutoinstProfile
+    # This class represents an AutoYaST <networking> section
```

Comment:
```suggestion
    # This class represents an AutoYaST \<networking> section
```

this fixes the yard rendering

Reply:
How about:

```rb
# TODO so far this contains only RoutingSection.
# The other aspects (interfaces, DNS resolver) are done in legacy code,
# see {Yast::LanClass#Import}.
```

Reply:
well, I hope it is just intermediate state and we move forward with this changes.